openevent/src/main/java/com/wearhello/mananwason/openevent/MainActivity.java

```diff
@@ -0,0 +1,24 @@
+package com.wearhello.mananwason.openevent;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.support.wearable.view.WatchViewStub;
+import android.widget.TextView;
+
+public class MainActivity extends Activity {
```

Comment:
I believe there is a http://developer.android.com/reference/android/support/wearable/activity/WearableActivity.html which should be used.